membersrvc/ca/ca.go

```diff
@@ -340,6 +345,9 @@ func (ca *CA) createCertificate(id string, pub interface{}, usage x509.KeyUsage,
 }
 
 func (ca *CA) createCertificateFromSpec(spec *CertificateSpec, timestamp int64, kdfKey []byte) ([]byte, error) {
+	mutex.Lock()
```

Comment:
@adecaro Transactions in SQLite3 are serialized but would that ensures concurrent transactions are serialized entirely or could them be interleaved? Would that depends on the underlying database driver? If so I think the advantage of this approach (synchronization handled uniformly in go) would be being independent of how concurrent transactions are handled in the db and being able to have mutex logic at our application level instead. Does it makes sense?